cmd/pmm-managed/main.go

```diff
@@ -82,14 +82,17 @@ var (
 	dbUsernameF = flag.String("db-username", "pmm-managed", "Database username")
 	dbPasswordF = flag.String("db-password", "pmm-managed", "Database password")
 
-	agentMySQLdExporterF = flag.String("agent-mysqld-exporter", "/usr/local/percona/pmm-client/mysqld_exporter", "mysqld_exporter path")
-	agentQANBaseF        = flag.String("agent-qan-base", "/usr/local/percona/qan-agent", "qan-agent installation base path")
+	agentMySQLdExporterF    = flag.String("agent-mysqld-exporter", "/usr/local/percona/pmm-client/mysqld_exporter", "mysqld_exporter path")
+	agentRDSExporterF       = flag.String("agent-rds-exporter", "/usr/sbin/rds_exporter", "rds_exporter path")
```

Comment:
Why rds_exporter is under `/usr/sbin` but mysqld_exporter under `/usr/local/percona/pmm-client`?

Reply:
rds_exporter is not a part of pmm-client distribution package.

Reply:
This still doesn't explain why we are not pushing it under the same dir. Not that this is a big issue anyway.